dashboard/config/locales/en.yml

```diff
@@ -711,9 +711,9 @@ en:
   flex_category:
     required: 'Overview'
```

Comment:
I'd prefer to keep it 'required' - required is the module type. it may be different in other sections